### PR TITLE
Cybernetics mod

### PR DESCRIPTION
**Pull request by Rodziel**
Added cybernetics mod. Can be toggled.
When mod is on, to equip prosthetics it is necessary to purchase/build interface and prosthetics, add interface to slave through surgery and equip prosthetics through cybernetics management screen. 
Added ocular implants. Slaves having them need to configure their eyes through cybernetics instead of salon.
Research lab in accessible through arcology management to research/build prosthetics and other tech. Fully functional if somewhat light on flavor text.

Other edits:
Added conditional to body mod studio so you can't brand non-existent body parts.
Added eugenics filter to recruiter. If any eugenics SMR are active it will extend recruitment period but recruits will be SMR compliant(+6 per SMR).
Added surgery to remove eyes rather than simply blind slaves(.eyes == -3). Currently unfinished and commented out.
Added PLimb, origEye and origHColor to slave variables and edited slave generators to include them. I'm not sure where to patch slaves that get imported from database.
PLimb tracks prosthetics interface, origEye checks for original eye color when equipping lenses except when set to "none" or "implant", origHColor tracks original hair color except when set to "bald".
Added wigs to salon for when permanent hair removal is added. 
Added hair removal in surgery/description.

Removed extra variables other than genes. Some of the descriptions could benefit from it.
Was only adding them so I don't have to wait for next major update to start adding things.
Keeping a fork conflict free is a pain in the ass.

### CODE REVIEW

**Comment by 000-250-006** [comment]
yay Gynoids!

**Comment by freecitiesdev** [comment]
Great work.

Feel free to continue adding to this until at least Monday 3/20/17, Since adding this many vars will definitely require NGP, this'll go in with 0.9.11, not before.

**Comment by Folant** [comment]
https://github.com/Rodziel/Free-Cities/commit/99164b1572f793cd462c883a67f30c2e84beb36a#diff-80d4ee5fa1b603a349f444b3c6c85a09 would probably revert https://github.com/Free-Cities/Free-Cities/commit/a602f326b875c784c9e17084e4c21f48709eb59c#diff-80d4ee5fa1b603a349f444b3c6c85a09

File: src/uncategorized/scheduledEvent.tw [hunk 1]
```diff
@@ -104,7 +104,9 @@
 <<elseif ($birthee != 0)>>
 	<<set $birthed to 0>>
 	<<goto "SE Birth">>
-<<elseif ($recruiterProgress >= 13)>>
+<<elseif ($recruiterEugenics == 0 and $recruiterProgress >= 13>>
```
Comment (Folant) [inline src/uncategorized/scheduledEvent.tw]: Missing `)`

File: src/uncategorized/scheduledEvent.tw [hunk 2]
```diff
@@ -104,7 +104,9 @@
 <<elseif ($birthee != 0)>>
 	<<set $birthed to 0>>
 	<<goto "SE Birth">>
-<<elseif ($recruiterProgress >= 13)>>
+<<elseif ($recruiterEugenics == 0 and $recruiterProgress >= 13>>
+	<<goto "SE recruiter success">>
+<<elseif ($recruiterEugenics == 1 and $recruiterProgress >= (13 += ($IntelligenceEugenicsSMR+$HeightEugenicsSMR+$FaceEugenicsSMR)*6))>>
```
Comment (Folant) [inline src/uncategorized/scheduledEvent.tw]: Should be `...$recruiterProgress >= (13 + (...`

**Comment by freecitiesdev** [comment]
I have some minor edits to request.

Can we strip out all the added variables that aren't used in this mod? Experience leads me to hold off on adding things like .urethra and .labiaMj until there's actual content that requires them.

**Pull request closed without merging**